sdk/core/core-rest-pipeline/review/core-rest-pipeline.api.md

```diff
@@ -263,6 +303,9 @@ export interface RestErrorOptions {
     statusCode?: number;
 }
 
+// @public
+export function retrieveToken(options: ChallengeCallbackOptions): Promise<AccessToken | undefined>;
```

Comment:
This also feels strange to have in this minimal package and the name is rather broad, since there are many tokens in the world

Reply:
This method has been removed. `getToken()` has been renamed to `getAccessToken()`